addons/website_sale/models/website_snippet_filter.py

```diff
@@ -13,21 +15,21 @@ def _get_website_currency(self):
 
     def _get_hardcoded_sample(self, model):
         samples = super()._get_hardcoded_sample(model)
-        if model and model.model == 'product.product':
+        if model is not None and model._name == 'product.product':
```

Comment:
None is Falsy...

why compare is Not None ?

Reply:
Because empty list model is also falsy

Reply:
yes, but in which case model will be a [] in the current code ? 
If I'm not wrong, it will be always filter.model.name or filter.action_id.model

And it is required to have at least one of them

Reply:
Right, it can never be None